api-catalog-ui/frontend/src/components/DetailPage/DetailPage.jsx

```diff
@@ -202,7 +211,7 @@ export default class DetailPage extends Component {
                                             className="links"
                                             onClick={(e) => this.handleLinkClick(e, '#tutorials-label')}
                                         >
-                                            Tutorials ({tutorialsCounter})
+                                            TechDocs Resources ({tutorialsCounter})
```

Comment:
is it ok to have this text here? Should it be custom?

Reply:
I think it's fine, it doesn't contain any specific reference